app/controllers/account/admin/categories_controller.rb

```diff
@@ -0,0 +1,94 @@
+# Encoding: utf-8
+module Account
+  module Admin
+    class CategoriesController < ApplicationController
+      before_action :admin_or_moderator?
+      before_action :status_inspector, only: [:show]
+      before_action :find_category, except: [:index, :create, :new]
+
+      def index
+        @categories = Category.all
+        smart_listing_create :categories, @categories,
+                             sort_attributes: [
+                               [:created_at, 'categories.created_at'],
+                               [:updated_at, 'categories.updated_at'],
+                               [:name, 'categories.name'],
+                               [:calculate, 'categories.issues_count']
+                             ],
+                             default_sort: { name: 'asc' }, partial: 'category'
+      end
+
+      def show
+        @status = params[:status] || 'opened'
```

Comment:
@denys281, updated info - smart_listing gem needs it to be global, otherwise, search doesn't work.